pkg/util/iptables/iptables_test.go

```diff
@@ -35,7 +35,12 @@ import (
 	fakeexec "k8s.io/utils/exec/testing"
 )
 
-const TestLockfilePath = "xtables.lock"
+func getLockPaths() (string, string) {
+	lock14x := fmt.Sprintf("@xtables-%d", time.Now().Nanosecond())
+	lock16x := fmt.Sprintf("xtables-%d.lock", time.Now().Nanosecond())
+	// The abstract socket and lock file will be removed by calling `runner.Close()`
```

Comment:
There is no `runner.Close()` method. They get removed by `restoreInternal` whenever they're successfully created. I don't think there needs to be any comment about this here though. So just remove this, then lgtm.

Reply:
Oh it is a typo.. I mean `locker.Close()`. Will remove it.